tools/src/main/java/org/apache/kafka/tools/GetOffsetShell.java

```diff
@@ -132,7 +132,7 @@ public GetOffsetShellOptions(String[] args) throws TerseException {
                     .ofType(String.class);
             timeOpt = parser.accepts("time", "timestamp of the offsets before that. [Note: No offset is returned, if the timestamp greater than recently committed record timestamp is given.]")
                     .withRequiredArg()
-                    .describedAs("<timestamp> / -1 or latest / -2 or earliest / -3 or max-timestamp")
+                    .describedAs("<timestamp> / -1 or latest / -2 or earliest / -3 or max-timestamp / -4 or earliest-local-timestamp")
```

Comment:
Is it possible to format this so it's easier to visually parse?

In the KIP, it looks like this:

```
--time <String: <timestamp> / -1 or        timestamp of the offsets before that.
  latest / -2 or earliest / -3 or max-     [Note: No offset is returned, if the
  timestamp / -4 or earliest-local /       timestamp greater than recently
  -5 or latest-tiered                      committed record timestamp is
                                           given.] (default: latest)
```

Is it worth the effort/hassle to make it look like this:

```
--time <timestamp> /                       timestamp of the offsets before that.
  -1 or latest /                           [Note: No offset is returned, if the
  -2 or earliest /                         timestamp greater than recently
  -3 or max-timestamp /                    committed record timestamp is
  -4 or earliest-local /                   given.] (default: latest)
  -5 or latest-tiered
````